Ecosystem Partners/Marketplace Guides/getting-started-with-a10-vthunder-partner-template.md

```diff
@@ -7,13 +7,14 @@
 }}}
 
 ![logo](http://www.a10networks.com/images/logo_s.png)
-###Partner Profile
+
+### Partner Profile
 A10 Networks - An industry leader in data security and cloud security solutions.
 
-http://www.A10Networks.com
+[http://www.A10networks.com](http://www.a10networks.com)
 
 #### Contact A10
-#####A10 Sales and Support:
+##### A10 Sales and Support:
 - Email Support - support@a10networks.com
 - Web Support - http://www.a10networks.com/support/
```

Comment:
[http://www.a10networks.com/support/](http://www.a10networks.com/support/)

since you went to the trouble of converting all the rest of them.